tests/v3_api/common.py

```diff
@@ -221,6 +221,16 @@ def validate_workload_with_sidekicks(p_client, workload, type , ns_name, pod_cou
         assert "running" in pod["status"]["containerStatuses"][1]["state"]
 
 
+def validate_workload_image(p_client, workload, expectedImage):
+    image = p_client.list_workload(workload=workload.id)[0].containers[0].image
+    assert image == expectedImage
+
+
+def validate_workload_paused(p_client, workload, expectedStatus):
+    workloadStatus = p_client.list_workload(uuid=workload.uuid)[0].paused
+    assert workloadStatus == expectedStatus
```

Comment:
@adingilloRancher Make sure that the pods of the workloads are in the right expected state.